Library/UIBarButtonItem.swift

```diff
@@ -2,7 +2,7 @@ import class UIKit.UIBarButtonItem
 import struct ObjectiveC.Selector
 
 public extension UIBarButtonItem {
-  public static func back(_ target: AnyObject?, selector: Selector) -> UIBarButtonItem {
+  public static func back(_ target: AnyObject?, selector: Selector?) -> UIBarButtonItem {
```

Comment:
should the selector be optional for a bar button? that seems like potential for bad UX.